fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/GenericDataServiceImpl.java

```diff
@@ -169,7 +170,8 @@ public String generateJsonFromGenericResultsetData(final GenericResultsetData gr
                             writer.append(
                                     "[" + localDate.getYear() + ", " + localDate.getMonthValue() + ", " + localDate.getDayOfMonth() + "]");
                         } else if (currColType.equals("DATETIME")) {
-                            final LocalDateTime localDateTime = LocalDateTime.parse(currVal);
+                            final DateTimeFormatter simpleDateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
```

Comment:
Let's extract this into a constant and name the variable what it represent. We don't need to waste an object creation every time we reach this execution path.

Reply:
Done!